packages/calcite-components/src/components/notice/notice.tsx

```diff
@@ -160,7 +175,7 @@ export class Notice
     const hasActionEnd = getSlotted(el, SLOTS.actionsEnd);
 
     return (
-      <div class={CSS.container}>
+      <div class={CSS.container} ref={this.setTransitionEl}>
```

Comment:
Can you disable `react/jsx-sort-props` here with the explainer from https://github.com/Esri/calcite-design-system/pull/7584?